src/components/personal-information/index.js

```diff
@@ -194,6 +224,68 @@ const PersonalInfoComponent = ({
                                     </div>
                                 }
 
+                                {formValues.ticketQuantity === 1 &&
```

Comment:
please enclose this on a method that gives some context like
shoudShowAlternateFlow and explain a little the rationale of that using comments on code for further maintenance